metau-capstone/View Controllers/StudyViewController.m

```diff
@@ -42,15 +42,14 @@ - (void)viewDidLoad {
     PFUser *const user = [PFUser currentUser];
     self.prevFinishedDate = user[@"prevFinishedDate"];
     
-    self.percentFinished = [user[@"percentFinished"] doubleValue];
-    [self.circleProgressBar setProgress:self.percentFinished animated:YES];
-    
     if ([self isFirstTimeUser] || [self isNewDay]) {
         // Check user has started reviewing for the day
         if (![self isFirstTimeUser] && [user[@"didStartReview"] isEqual:@NO]) {
             // Increment day counter for the user
             [user incrementKey:@"userDay"];
             [user saveInBackground];
+            // Reset progress bar to zero percent
+            user[@"percentFinished"] = @(0);
```

Comment:
Does this get stored in parse? Or does it just live on the user object?

Reply:
Ah, I looked more closely in my code and I call [user saveInBackground]; later on which is why I think the percentFinished value was also saved in Parse. To decrease the number of [user saveInBackground]; calls, I deleted both current occurrences and call it once at the end of viewDidLoad. Thanks for catching this! And let me know if this way doesn't seem better.